ArdusatSDK.cpp

```diff
@@ -364,8 +369,8 @@ int calculateCheckSum(const char *sensor_name, int num_vals, ...) {
 /*
  * toCSV Output functions
  */
-void _output_buffer_reset() {
-  memset(_output_buffer, 0, 512);
+void _resetOutBuf() {
+  memset(_getOutBuf(), 0, OUTPUT_BUF_SIZE);
```

Comment:
You probably want to assign the output buffer to a pointer in this function, when repeating the same call so much - it is relatively expensive (given the size of the callee function).